extra-multimedia/vapoursynth/autobuild/beyond

```diff
@@ -0,0 +1 @@
+sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
```

Comment:
This is unnecessary for AOSC OS. Before you copy from Arch Linux, think carefully about what each line does.

Reply:
This is indeed something that I didn't think about carefully, so I won't take it as an example.

Reply:
Dropped this in 145529b.